src/libraries/System.Threading.Tasks.Dataflow/tests/Dataflow/DataflowBlockExtensionTests.cs

```diff
@@ -1121,6 +1121,7 @@ public async Task TestReceiveAsync_ManyInOrder()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // uses a lot of stack
```

Comment:
Were this and `TestOutputAvailableAsync_LongSequence` the only ones to use a lot of the stack? Were there other indications that led to skipping the test with `PlatformSpecific` rather than `ConditionalFact`?

Reply:
Yeah and to be honest they didn't really show up during my testing on wasm but from a conversation I saw on Slack where @BrzVlad recommended we disable this test when using the interpreter so I figured wasm is in the same boat: https://xamarinhq.slack.com/archives/C03CELHQQ/p1593621506218500 :smile: